target/supervisor/conf.d/supervisor-app.conf

```diff
@@ -96,6 +96,24 @@ stdout_logfile=/var/log/supervisor/%(program_name)s.log
 stderr_logfile=/var/log/supervisor/%(program_name)s.log
 command=/usr/sbin/amavisd-new foreground
 
+[program:rspamd]
+startsecs=0
+stopwaitsecs=55
+autostart=false
+autorestart=true
+stdout_logfile=/var/log/supervisor/%(program_name)s.log
+stderr_logfile=/var/log/supervisor/%(program_name)s.log
+command=/usr/bin/rspamd --no-fork --user=rspamd --group=rspamd
+
+[program:redis]
+startsecs=0
+stopwaitsecs=55
+autostart=false
+autorestart=true
+stdout_logfile=/var/log/supervisor/%(program_name)s.log
+stderr_logfile=/var/log/supervisor/%(program_name)s.log
+command=redis-server --daemonize no --bind 127.0.0.1 --port 6379 --loglevel warning
```

Comment:
I don't use redis, so just noticed that this port differed from the `redis.conf`, is that correct?

Reply:
The port number is correct here :) It is the default here.